README.md

```diff
@@ -96,3 +96,32 @@ Import `connect` function from our `./store` and use it for getting state and di
 
 <button on:click="{increment}">+</button>
 ```
+
+## Usage with [@storeon/router](https://github.com/storeon/router)
+If you want to use the @storeon/svelte with the [@storeon/router](https://github.com/storeon/router) you should import the `router.createRouter` from [@storeon/router](https://github.com/storeon/router) and add this module to `createSvelteStore` instead of `createStore`
+
+#### `store.js`
+```js
+import { createSvelteStore } from "@storeon/svelte";
+import router from '@storeon/router'
+const connect = createSvelteStore([
+  router.createRouter([
+    ['/', () => ({ page: 'home' })],
+    ['/blog', () => ({ page: 'blog' })],
+  ])
+])
+```
+
+And use it like:
+#### `App.svelte`
+```svelte
```

Comment:
does github recognize `svelte` as syntax highlight? maybe better to use `html`?

Reply:
@distolma Yeah, github recognizes it pretty well. You can take a look at [the blob of this file](https://github.com/storeon/svelte/blob/d9c931683ac404bf9fdd09f3c252ab9891a1316b/README.md)

Reply:
@distolma It correctly add `highlight highlight-source-svelte` classes to the div with this code:)